modules/local/locidex_search.nf

```diff
@@ -0,0 +1,56 @@
+/* Locidex search function for the allele calling
+
+*/
+
+process LOCIDEX_SEARCH {
+
+    tag "$meta.id"
+    label "process_high"
```

Comment:
How many resources does `locidex search` need? If it's not too many, could this be lowered to `process_medium` or `process_low`?

Reply:
I beleive this process benefits from threading and can be quite slow as its is utilizing blast and alignmnents. Process medium would probably work better for this

Reply:
switched to medium in: c8e0c8a003520471478e446d9fef5107084608ea